cassandra-persistence/src/main/java/com/netflix/conductor/cassandra/config/CassandraProperties.java

```diff
@@ -72,17 +75,20 @@
     /**
      * The time in seconds after which the in-memory task definitions cache will be refreshed
      */
-    private int taskDefCacheRefreshTimeSecs = 60;
+    @DurationUnit(ChronoUnit.SECONDS)
```

Comment:
Is the DurationUnit still needed? We could update the documentation to say that Duration format is supported.